lib/heartcheck/controllers/environment.rb

```diff
@@ -0,0 +1,17 @@
+require 'sys-uname'
+
+module Heartcheck
+  module Controllers
+    class Environment < Base
+      def index
+        MultiJson.dump(
+          {
+            'system-info' => Sys::Uname.uname,
```

Comment:
The return of the method `Sys::Uname.uname` is a Struct Object:
```ruby
irb(main):009:0> Sys::Uname.uname
=> #<struct Struct::UnameStruct sysname="Linux", nodename="perrellalw", release="4.4.0-96-generic", version="#119-Ubuntu SMP Tue Sep 12 14:59:54 UTC 2017", machine="x86_64", domainname="(none)">
irb(main):010:0>
```

What about using `.to_h` like this:

```ruby
irb(main):008:0> Sys::Uname.uname.to_h
=> {:sysname=>"Linux", :nodename=>"perrellalw", :release=>"4.4.0-96-generic", :version=>"#119-Ubuntu SMP Tue Sep 12 14:59:54 UTC 2017", :machine=>"x86_64", :domainname=>"(none)"}
```

Reply:
Great idea, I just pushed this change, along with the tests